### DISCUSSION TITLE
Why commands that I used in bash is not recognized by the nushell ?

### DISCUSSION THREAD

**Discussion by ilia4bit** [nushell/nushell#7601] [Q&A] [answered]
Hey, I'm new to Nushell and it's awesome 🤩.
some commands that aren't built-in, don't work. like `hx`, `poetry`, `pipx`, `cargo`, `rustup`, ...
Is Nushell's PATH different from other shells PATH  variable ?
What should I do? Should I add their path to $PATH?
![Screenshot from 2022-12-25 16-16-07](https://user-images.githubusercontent.com/69800165/209468458-82227926-2276-46db-a9ce-0950ce6dffe8.png)

**Comment by fdncred** [comment, marked as answer]
You need to add folders to the $env.PATH in the env.nu file https://www.nushell.sh/book/environment.html#environment

**Reply by ilia4bit** [reply]
Thanks 🙏🏼
